PactNet.Tests/PactVerifierConfigTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PactNet.Tests
+{
+    public class PactVerifierConfigTests
+    {
+        [Fact]
+        public void CustomHeader_backwardCompatibility_getValue()
+        {
+            var config = new PactVerifierConfig();
+
+            KeyValuePair<string, string> keyValuePair = new KeyValuePair<string, string> ( "key name", "value");
+
+#pragma warning disable CS0618 // Type or member is obsolete
+            config.CustomHeader = keyValuePair;
+
+            Assert.Equal(config.CustomHeader, keyValuePair);
```

Comment:
because there is a line of a code, rather than a plain set/get, there is potential for the code to be broken from a change. I chose to include this to catch breaking changes before we send out a breaking change to the various packages we are using